### PR TITLE
Implement optional frameskip feature.

### PR DESCRIPTION
**Pull request by littleguy77**
On mobile hardware, this makes a big difference in playability on most ROMs.  E.g. on the Tegra3-based Nexus 7 (2012) and OUYA, games like DK64 and Diddy Kong Racing are not playable without this feature.

I understand that frameskip is blasphemy in some communities, so this feature is only enabled via build flag.  In other words, by default, this PR makes no change whatsoever to the compiled binary.

@paulscode

### CODE REVIEW

File: projects/unix/Makefile [hunk 1]
```diff
@@ -424,6 +432,7 @@ targets:
 	@echo "    NO_ASM=1      == build without inline assembly code (x86 MMX/SSE)"
 	@echo "    USE_GLES=1    == build against GLESv2 instead of OpenGL"
 	@echo "    NO_SSE=1      == build without SSE support"
+	@echo "    USE_FRAMESKIPPER=1 == build with frameskipper feature"
 	@echo "    APIDIR=path   == path to find Mupen64Plus Core headers"
```
Comment (Narann) [inline projects/unix/Makefile]: I tend to prefer one commit per feature as most diff tools allow you to skip space change to focus on code change (I know I know, my combiner refactoring does not follow this statement...). But I don't see any _real_ problem to do cosmetic changes in separate commits. :)

**Comment by richard42** [comment]
I think the 'clock_gettime' function is unavailable in Win32.  Can you provide an alternate getCurrentTicks functions so that this change will compile in Visual Studio?

**Comment by littleguy77** [comment]
Ok, I will take a look and update the PR when I have a portable clock solution.

**Comment by littleguy77** [comment]
Ok, just updated the PR.  Frameskipper now uses `SDL_GetTicks()` for cross-platform portability.
